front/app/components/Form/Components/Controls/MatrixControl.tsx

```diff
@@ -185,11 +189,7 @@ const MatrixControl = ({
                   {columnsFromSchema.map((_, columnIndex) => {
                     return (
                       <StyledTd key={`radio-${columnIndex}-${index}`}>
-                        <Box
-                          min-width="84px"
```

Comment:
`min-width` doesn't work on Box (needs to be `minWidth`), so this wasn't doing anything.